gnomad/resources/grch37/gnomad.py

```diff
@@ -156,6 +156,22 @@ def _public_browser_gene_ht_path() -> str:
     return "gs://gnomad-public-requester-pays/resources/grch37/browser/gnomad.genes.GRCh37.GENCODEv19.pext.ht"
 
 
+def _public_mnv_path(distance: str = "1") -> str:
+    """
+    Get public multinucleotide variant (MNV) data.
+
+    :param distance: Distance between two SNVs in MNV. Default is '1'.
+    :return: Path to MNV data.
+    :raises DataException: If the provided distance is invalid (outside 1-10).
+    """
+    distances = set(map(str, range(1, 10)))
+    if distance not in distances:
+        raise DataException(
+            f"Invalid distance: '{distance}'. Valid options are {distances}."
+        )
+    return "gs://gnomad-public-requester-pays/release/2.1/mnv/gnomad_mnv_genome_d{distance}.ht"
```

Comment:
are there supposed to be files here? it looks like there should be another subdir in the path, although there's both "genome" and "genomes"?

Reply:
ah oops you're right --  I missed "genome" in the path. not sure why we have "genome" and "genomes", but the downloads page points to "genome" so will put that one here